qupath-core/src/main/java/qupath/lib/color/ColorMaps.java

```diff
@@ -516,8 +497,25 @@ public Integer getColor(double value, double minValue, double maxValue) {
 //		}
 
 	}
-	
-	
+
+	/**
+	 * Utility method to translate values to indices, handling swapped min/max values.
+	 * @param value The value to be mapped.
+	 * @param minValue The min value of the color scale.
+	 * @param maxValue The max value of the color scale.
+	 * @param nColors The number of colors in the color scale.
+	 * @return An integer to index the colors.
+	 */
+	static int getInd(double value, double minValue, double maxValue, int nColors) {
+		int ind = 0;
+		double max = Math.max(minValue, maxValue);
+		double min = Math.min(minValue, maxValue);
+		ind = (int)((value - min) / (max - min) * nColors + .5);
+		ind = ind >= nColors ? nColors - 1 : ind;
+		ind = Math.max(ind, 0);
+		if (minValue > maxValue) ind = (nColors - 1) - ind;
```

Comment:
Tend to agree, though I also disfavour braceless ifs which are used more broadly

Reply:
Fair, happy to be convinced to try to move away from braceless ifs.

Seems to be consistent with the Google style guide too: https://google.github.io/styleguide/javaguide.html#s4.1-braces

Reply:
There's nothing on it in the style guide, but I personally always use new lines with brackets even if the code inside the `if` statement is very short.

What do you think?

Reply:
But for this specific case (assigning a value) I'd use a ternary operator instead (like on line 514)

Reply:
Hmm, I'm slow to ternaries (mainly because some languages don't have them, but if I was doing one here it'd be `minValue > maxValue ? (nColors - 1 - ind) : ind;` to make the distinction between elements of the ternary clearer